presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergConfig.java

```diff
@@ -43,6 +43,7 @@ public class IcebergConfig
     private double minimumAssignedSplitWeight = 0.05;
     private boolean parquetDereferencePushdownEnabled = true;
     private boolean mergeOnReadModeEnabled;
+    private boolean pushdownFilterEnabled;
```

Comment:
The commit title "Add iceberg.pushdown-filter-enabled config property to control Filter…
… Pushdown in Iceberg Connector" is too long. "Add iceberg.pushdown-filter-enabled config property" would be just good.